docs-website/sidebars.js

```diff
@@ -249,6 +249,11 @@ module.exports = {
               id: "docs/managed-datahub/slack/saas-slack-setup",
               className: "saasOnly",
             },
+            {
+              type: "doc",
+              id: "docs/managed-datahub/slack/saas-slack-troubleshoot",
+              className: "saasOnly",
+            },
```

Comment:
![image](https://github.com/user-attachments/assets/a3979d57-a161-459c-9233-e55081de57b8)
maybe this should go to the bottom?

Reply:
good call